bash_command_line_102/bash_command_line_102.md

```diff
@@ -269,12 +299,14 @@ Now you can use `cd` and `ls` to see that `blue_jay.txt` is now in `blue_animals
 
 Notice that we used the **relative** locations of `blue_animals` and the `blue_jay.txt` file. We could also have used their global location. Let's see how we would use **global** locations to move the `blue_morpho.txt` file, currently in `blue_animals`, to the same location as the other animals.
 
+Tip: You can scroll sideways to see all of the text in the code block:
```

Comment:
I just have a really minor suggestion of possibly making the tip text bold.
```suggestion
**Tip: You can scroll sideways to see all of the text in the code block:**
```

Reply:
Great idea! Thanks!